lib/Distribution/Opensuse/Leap/15.pm

```diff
@@ -8,7 +8,7 @@
 # It follows latest SLE 15 and only if it behaves different in Leap 15 then it
 # should be overriden here.
 #
-# Maintainer: QE YaST <qa-sle-yast@suse.de>
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam@suse.de>
```

Comment:
why not just use simple key work  "QE Yam team" instead of "QE YaST and Migration (QE Yam)"

Reply:
In ticket, it advises to use "QE YaST and Migration (QE Yam)", I think in case of others don't know what does "QE Yam" mean. Using "QE YaST and Migration (QE Yam)" makes it more clear.